tests/lblk.c

```diff
@@ -50,14 +50,14 @@ boilerplate(struct xnvmec *cli, uint8_t **wbuf, uint8_t **rbuf, size_t *buf_nbyt
 
 	*wbuf = xnvme_buf_alloc(dev, *buf_nbytes);
 	if (!*wbuf) {
-		err = -errno;
+		err = -ENOMEM;
```

Comment:
No it is because scan-build assumes that errno could be 0 here, which would cause problems when we later try to use wbuf or rbuf.
Of course, we know it could never be 0 if the condition is true.